.github/actions/store_artifacts/action.yml

```diff
@@ -5,17 +5,29 @@ inputs:
     description: "Multi-line list of artifacts to store"
     required: true
   storage_dir:
-    description: "Directory name to store artifacts in"
+    description: "Subdirectory name for stored artifacts, same as product type"
     required: true
   storage_root:
-    description: "Root path of the storage to place artifacts to"
+    description: "Root path to place artifacts to"
     required: true
+  event_name:
+    description: "GitHub event name. If not set - taken from github context"
+    required: false
+  branch_name:
```

Comment:
do we need the both branch_name and trigger_repo_sha? I think sha can be used as branch because it should be unique.

Reply:
We need branch name to store artifacts in the right folder

Reply:
As discussed offline, this is a technical limitation of the old approach to save artifacts in  the branch subfolders, will be easier to find them using branch names too. Closing